app/interceptors/ExceptionInterceptor.java

```diff
@@ -72,7 +71,7 @@ private int getStatusCode(final Throwable throwable) {
     private String getMessage(final Throwable throwable, final int status) {
         String message = throwable.getMessage();
         if (StringUtils.isBlank(message)) {
-            message = HttpStatus.getStatusText(status);
+            message = Integer.toString(status);
```

Comment:
I think those are transitive dependencies that are already there, and don't have to be added (just check Eclipse). But I would not add them just for this if they aren't.